src/gc/env/gcenv.base.h

```diff
@@ -320,6 +320,11 @@ inline void* ALIGN_DOWN(void* ptr, size_t alignment)
     return reinterpret_cast<void*>(ALIGN_DOWN(as_size_t, alignment));
 }
 
+inline int GetRandomInt(int max)
```

Comment:
Why not call `rand()` directly? This is used just in one place, in debug-only code.

Reply:
I heeded this warning that I got when I tried to do that:

```
e:\github\coreclr\src\gc\handletablecore.cpp(1054): error C3861: 'Do_not_use_rand': identifier not found [E:\GitHub\co reclr\bin\obj\Windows_NT.x64.Debug\src\vm\wks\cee_wks.vcxproj]
```

I can always undef `Do_not_use_rand` but it seemed to go against the spirit of the warning 😄 .